lib/main.coffee

```diff
@@ -34,6 +34,7 @@ module.exports =
   consumeFileIcons: (service) ->
     FileIcons.setService(service)
     @fileIconsDisposable = service.onWillDeactivate -> FileIcons.resetService()
```

Comment:
Shouldn't we call `@treeView?.updateRoots()` from within the `service.onWillDeactivate` callback as well?